libs/model/src/test/java/de/cyface/model/MeasurementTest.java

```diff
@@ -231,6 +241,135 @@ void testWriteMeasurementAsJson() {
         assertThat(jsonOutput.toString(), is(equalTo(expectedOutput)));
     }
 
+    /**
+     * Ensures track buckets are sorted before they are composed to a Track.
+     */
+    @Test
+    void testTracks_toSortBuckets() throws ParseException {
```

Comment:
These tests moved here from `collector/libs/api/MeasurementIterator` as the methods under test moved to `libs/model.Measurement` in this repo.